scripts/connlimit.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
would `scripts/mysqlconnlimit.sh` be a better name?

Reply:
Yep, done.